tutorials/scripting/gdscript/gdscript_exports.rst

```diff
@@ -294,13 +293,6 @@ Packed type arrays also work, but only initialized empty:
     @export var vector3s = PackedVector3Array()
     @export var strings = PackedStringArray()
 
-Default value can include run-time values, but can't
-be exported.
-
-::
-
-    var c = [a, 2, 3]
-
```

Comment:
I removed this part as it's not an exported variable, so listing it here can be misleading.